.github/workflows/format.yml

```diff
@@ -0,0 +1,52 @@
+name: Format
+on: [pull_request]
+
+jobs:
+  format-python:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ./python/coinbase-agentkit
```

Comment:
Formatting should be checked in all packages instead of just `coinbase-agentkit`

Reply:
@John-peterson-coinbase @stat let's merge this as-is, and we will fast follow with this, which we should also apply the to lint action.